### PR TITLE
structure: Don't add ID prefixes to SEO-relevant elements

### PR DESCRIPTION
**Pull request by ghost**
As discussed in, uh, March 2021: https://confluence.suse.com/x/rgEYKQ

### CODE REVIEW

**Review by jfaltenbacher — APPROVED** [review]
LGTM

**Review by janajaeger — APPROVED** [review]
LGTM

**Review by PadraigDillon1 — APPROVED** [review]

File: xml/docu_styleguide.structure.xml [hunk 1]
```diff
@@ -1259,10 +1259,18 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
       <formalpara>
        <title>Prefix</title>
        <para>
-        Signifies the type of XML element. Use in accordance with
-        <xref linkend="tab-id-prefix"/>.
+        Signifies the type of XML element.
+        Prefixes are meant as an aid to writers to allow creating
+        logically-named, but distinct identifiers even for closely related
+        elements.
```
Comment (cwickert) [inline xml/docu_styleguide.structure.xml:1265]: Simplify to
```suggestion
        Prefixes aid writers with creating logically-named, but distinct
        identifiers for elements.
```

**Review by cwickert — APPROVED** [review]
LGTM except for one simplification.

Otherwise we can just merge as-is and then simplify the whole section later. It's just written _way_ to complicated.

File: xml/docu_styleguide.structure.xml (hunk 1 above)
Comment (dariavladykina) [inline xml/docu_styleguide.structure.xml:1265]: Prefixes aid writers in creating logically named but distinct identifiers for elements.

Comment (cwickert) [reply xml/docu_styleguide.structure.xml:1265]: We could even simplify further:
```suggestion
        Prefixes aid writers in creating logically named identifiers for elements.
```

**Pull request merged**